app/services/account_reset/delete_account.rb

```diff
@@ -39,6 +39,7 @@ def handle_successful_submission
     end
 
     def destroy_user
+      Db::DeletedUser::Create.call(user.id)
```

Comment:
should we consider doing this in a transaction? so if one step fails we're not left in a half-state?

Reply:
done